src/Controls/src/Core/Shell/Shell.cs

```diff
@@ -1562,12 +1921,15 @@ protected override void OnPropertyChanged([CallerMemberName] string propertyName
 		#region Shell Flyout Content
 
 
-		/// <summary>Bindable property for <see cref="FlyoutContent"/>.</summary>
-
+		/// <summary>
+		/// Flyout items, which represent the flyout content, can optionally be replaced with your own content.
+		/// </summary>
 		public static readonly BindableProperty FlyoutContentProperty =
 			BindableProperty.Create(nameof(FlyoutContent), typeof(object), typeof(Shell), null, BindingMode.OneTime, propertyChanging: OnFlyoutContentChanging);
 
-		/// <summary>Bindable property for <see cref="FlyoutContentTemplate"/>.</summary>
+		/// <summary>
+		/// The flyout content can be defined by setting a DataTemplate.
```

Comment:
I think we should be more explicit as to what this property defines. At the moment I think this definition is a bit incomplete - its stating what _can_ be done, but doesn't really define the property itself _is_